openid-4-verifiable-presentations-1_0.md

```diff
@@ -549,11 +555,276 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+# Verifiable Presentation Query Language {#vp_query}
+
+The Verifiable Presentation Query Language (VP Query) is a JSON-encoded query
+language that allows the Verifier to request the Wallet to present Verifiable
+Credentials that match the query. The Verifier MAY encode constraints on the
+combinations of credentials and claims that are requested. The Wallet evaluates the
+query against the Verifiable Credentials it holds and returns the Verifiable
+Presentations that match the query.
+
+A valid VP Query is defined as a JSON-encoded object with the following
+top-level properties:
+
+`expect_credentials`:
+: REQUIRED. A non-empty array of Credential Queries as defined in (#credential_query)
+that specify the requested Verifiable Credentials.
+
+`valid_credential_sets`:
```

Comment:
Why is this array optional? It's simpler for parsing purposes if this is always present.

Reply:
I think this is covered below in the "Selecting Credentials" section where omission means implicitly all credentials in the `expected_credentials` structure is expected to be returned. Text for clarity

> - If `valid_credential_sets` is not provided, the Verifier requests all
  Credentials in `expect_credentials` to be returned.

While I agree it might unify parsing rules I tend to prefer the approach from @danielfett as it keeps simple queries simpler from a syntax/expression/verbosity perspective.